src/views/Profile/PersonalInfo/RegionCityHandler.tsx

```diff
@@ -133,11 +133,11 @@ const RegionCityHandler = ({
               variant="outlined"
               className={classes.TextField}
               required
-              // eslint-disable-next-line
-              {...params}
               label={i18n.t(langTokens.admin.region)}
               name="region"
               onBlur={blurHandler}
+              // eslint-disable-next-line
```

Comment:
which exact ESLint rule does it disable?

telling in advance: disabling ESLing completely, either to line or to whole file, is never a good idea
if you really need that, just list rules you are about to disable:

```
// eslint-disable-next-line rule1,rule2,rule3
```

Reply:
my fault. fixed it.
unfortunately, this component accepts a lot of props and it is allowed in the rule to disable it for this reason.
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md